### PR TITLE
DOCS-1367: Update I2C attributes for mpu6050

### PR DESCRIPTION
**Pull request by sguequierre**
* Change to index
* Remove board attribute
* Add new picture

### CODE REVIEW

File: docs/components/movement-sensor/mpu6050.md [hunk 1]
```diff
@@ -113,8 +95,7 @@ Then remove and fill in the attributes as applicable to your movement sensor, ac
 <!-- prettier-ignore -->
 | Name                  | Type    | Inclusion    | Description |
 | --------------------- | ------- | ------------ | ----------- |
-| `board`               | string  | **Required** | The `name` of the [board](/components/board/) to which the device is wired. |
-| `i2c_bus`             | string  | **Required** | The `name` of the [I<sup>2</sup>C bus configured](/components/board/#i2cs) on your [board](/components/board/) wired to this device. |
+| `i2c_bus`             | string  | **Required** | The index of the [I<sup>2</sup>C bus configured](/components/board/#i2cs) on your [board](/components/board/) wired to this device. |
```
Comment (penguinland) [inline docs/components/movement-sensor/mpu6050.md:98]: It's no longer configured on your board. I'd rephrase this part some more. I liked the phrasing in https://github.com/viamrobotics/docs/pull/2297/files: "The index of the [I2C bus] on the board that the sensor is wired to."

**Review by penguinland — APPROVED** [review]
Beyond that, LGTM!

File: docs/components/movement-sensor/mpu6050.md (hunk 1 above)
Comment (penguinland) [reply docs/components/movement-sensor/mpu6050.md:98]: :exploding_head: This file uses `I<sup>2</sup>C`, while the one in https://github.com/viamrobotics/docs/pull/2297 just uses I2C! I don't know which one we prefer, but I think they should be consistent from one to the other.

**Review by npentrel — APPROVED** [review]
Lgtm % penguinland's comment

File: docs/components/movement-sensor/mpu6050.md (hunk 1 above)
Comment (sguequierre) [inline docs/components/movement-sensor/mpu6050.md:98]:
```suggestion
| `i2c_bus`             | string  | **Required** | The index of the [I<sup>2</sup>C bus](/components/board/#i2cs) on the [board](/components/board/) that your movement sensor is wired to. |
```
